pronotepy/ent.py

```diff
@@ -474,3 +474,37 @@ def monbureaunumerique(username, password):
     cookies = requests.utils.cookiejar_from_dict(
         requests.utils.dict_from_cookiejar(session.cookies))
     return cookies
+
+def ent_essonne(username, password):
+    """
+    ENT Essonne
+
+    Parameters
+    ----------
+    username : str
+        username
+    password : str
+        password
+
+    Returns
+    -------
+    cookies : cookies
+        returns the ent session cookies
+    """
+    # ENT / PRONOTE required URLs
+    ent_login = "https://www.moncollege-ent.essonne.fr/auth/login"
+    # Required Headers
+    headers = {
+        'connection': 'keep-alive',
+        'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:73.0) Gecko/20100101 Firefox/73.0'}
+
+    payload = {
+        'email' : username,
+        'password' : password,
+    }
+    # ENT Connection
+    session = requests.Session()
+
+    cookies = requests.utils.cookiejar_from_dict(requests.utils.dict_from_cookiejar(session.cookies))
+    response = session.post(ent_login, headers=headers, data=payload, cookies=cookies)
```

Comment:
This needs to be changed to 
```py
response = session.post(ent_login, headers=headers, data=payload)
```
since [L508](https://github.com/bain3/pronotepy/pull/60/files#diff-6203c7f8202b3248dae5e1f35189b24f2ab11db401c8428a27f052ffa6ba0e58R508) has been removed